src/Surfnet/StepupGateway/GatewayBundle/Controller/SecondFactorController.php

```diff
@@ -36,8 +36,7 @@
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 
 /**
- * @SuppressWarnings(PHPMD.CouplingBetweenObjects) -- Too many second factor types in one controller. See Pivotal:
- *     https://www.pivotaltracker.com/story/show/104104610
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
```

Comment:
Imo, it's nice to know the cause of the CouplingBetweenObjects warning. What's the rationale behind removing it? (I understand the link is dead.)